lib/iron_worker_ng/client.rb

```diff
@@ -313,6 +313,14 @@ def tasks_create(code_name, params = {}, options = {})
       OpenStruct.new(t)
     end
 
+    def tasks_bulk_create(code_name, params = [], options = {})
+      IronCore::Logger.debug 'IronWorkerNG', "Calling tasks.bulk_create_legacy with code_name='#{code_name}', params='#{params.to_s}' and options='#{options.to_s}'"
+
+      res = @api.tasks_bulk_create(code_name, params.is_a?(Array) ? params : params.to_json, options)
```

Comment:
Code looks good to me. How about passing just params without checking `params.is_a?(Array)`, if you pass string it will raise en error NoMethodError: undefined method `map!' for String